site/zenodo_rdm/legacy/serializers/schemas.py

```diff
@@ -321,8 +496,16 @@ def dump_state(self, result, original, **kwargs):
     @post_dump(pass_original=True)
     def dump_prereserve_doi(self, result, original, **kwargs):
         """Dump prereserved DOI information."""
+        provider = original["pids"]["doi"]["provider"]
+        recid = original["id"]
+        # For external DOIs, the prereserve_doi is hacked on the response.
+        # TODO confirm this  hack
```

Comment:
For external dois, the `prereserve_doi` is not defined. Therefore I added it here manually